react/src/layout/report.tsx

```diff
@@ -377,6 +391,7 @@ export class SimulationStartLayout extends Layout<SimulationStartConfig, {}> {
                             <Stack gap={2}>
                                 <span>{'Simulation Completed'}</span>
                                 <span>{`Elapsed time: ${elapsedTimeSeconds} ${'second' + (elapsedTimeSeconds !== 1 ? 's' : '')}`}</span>
```

Comment:
The good old "plural function". Again it seems like it would be better to modularize this. For something that runs for a few days, displaying n*86400 seconds is not very useful. This will come up so we'll want to be able to switch formats based on the data.

Reply:
Yes, I agree. This was a band-aid approach and I could really use any info you have on making things plural. English sucks so I wonder if if would be any better than `(n > 1) ? "seconds" : "second"`. Should this just be a function like `let plural = (n, plural, singular) => (n > 1) ? plural : singular`? Either way I need to make the time handling better here. Hours and minutes and the whole thing. Will update that on this PR.

Reply:
Is `(n > 1) => "seconds" : "second"` any worse than `plural(n, "seconds", "second")`? I feel like I am barely missing the path where we can abstract this. All I would be abstracting here is the condition of `n > 1`. It feels just as complex in either case. You still need to know the order of the params but you just replace `n > 1` with the name `plural` and add some import in the process.

Reply:
I was thinking something [like this](https://github.com/plurals/pluralize/blob/master/pluralize.js). Not necessarily this library, but something that is an abstraction for taking the word and pluralizing. Again, there's an argument for modularizing the strings from tables with a bit of business logic.